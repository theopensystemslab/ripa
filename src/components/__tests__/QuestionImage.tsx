import { cleanup, fireEvent, render, wait } from "@testing-library/react";
import * as React from "react";

import StatementImage from "../QuestionImage";

afterEach(cleanup);

describe("Statement Image Component", () => {
  it("should render snapshot when image is available", () => {
    const { asFragment } = render(<StatementImage src={"example.png"} />);
    expect(asFragment()).toMatchSnapshot();
  });
  it("should render snapshot when image is not available", () => {
    const { asFragment } = render(<StatementImage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
