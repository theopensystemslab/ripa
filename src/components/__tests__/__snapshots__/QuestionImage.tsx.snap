// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Statement Image Component should render snapshot when image is available 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-1"
  >
    <img
      alt=""
      src="example.png"
    />
  </div>
</DocumentFragment>
`;

exports[`Statement Image Component should render snapshot when image is not available 1`] = `<DocumentFragment />`;
